packages/asc-ui/src/components/Checkbox/Checkbox.tsx

```diff
@@ -40,13 +41,14 @@ const Checkbox = React.forwardRef<
     return (
       <CheckboxWrapperStyle
         {...{ className, disabled, focus, checked, error }}
+        // eslint-disable-next-line no-nested-ternary
         aria-disabled={disabled}
```

Comment:
Do we need to add this `aria-disabled` attribute here? Since the checkbox itself already gets the disabled state as well.